mm2src/mm2_err_handle/src/mm_error.rs

```diff
@@ -258,6 +258,7 @@ impl<E: NotMmError> MmError<E> {
             .iter()
             .map(|src| src.file)
             .rev()
+            .sorted()
```

Comment:
I don't think we should sort the tracing of the error path

Reply:
Didn't really check what the function was doing. Do we need to deduplicate paths? It doesn't really work without sorting..

Reply:
> Do we need to deduplicate paths?

It's deduplication of filenames, so if in utxo in this `mm2.lp_swap.utxo.rpc_client` there was 2 successive callers we would include it only once, but if `mm2.lp_swap.utxo.rpc_client.utxo` it's fine.

Reply:
Reverted `sort` and added added comments why `sort` isn't needed there.